ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh

```diff
@@ -74,6 +74,7 @@ fi
 
 cat >>"${SHARED_DIR}/install-config.yaml" <<EOF
 baseDomain: ${base_domain}
+fips: true
```

Comment:
Same as above.

Reply:
Same for this? https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/agent/vsphere/install/compact/cucushift-agent-vsphere-install-compact-workflow.yaml#L9